content/pages/sponsors.md

```diff
@@ -1,9 +1,12 @@
 Title: Sponsorship
+status: hidden
 
 Occasionally companies will help us run events by providing us with funding, currently we are working
 on the following projects:
 
-### Technical Workshop Series with BlackBerry
+
+## Technical Workshop Series 
+#### Brought to you in partnership with ![BlackBerry](http://csclub.uwaterloo.ca/~fhboxwal/bb_logo.jpg)
```

Comment:
We should be hosting this image ourselves.

I've added the image, so when we next deploy it should be hosted at: http://wics.uwaterloo.ca/images/bb_logo.jpg